content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,707 @@
+   :alt: An onboarding slide in edit mode, with the main fields highlighted.
+
+Share job positions
+===================
+
+In order to earn referral points, a user must first share a job position so the applicant can apply
+for the position. There are several ways to share job positions within the Referrals application,
+through the :ref:`View Jobs <referrals/view>` button and the :ref:`Email A Friend <referrals/email>`
+button.
+
+.. _referrals/view:
+
+View Jobs
+---------
+
+To see all job positions that are actively recruiting candidates, click the :guilabel:`View Jobs`
+button on the main Referrals dashboard. This presents all job positions, with each individual job
+presented in its own card. Each job position card contains the following information:
+
+- The title of the job position. This information is taken from the :guilabel:`Job Position` field
+  of the job form.
+- The number of positions being recruited. This information is taken from the :guilabel:`Expected
+  New Employees` field of the :guilabel:`Recruitment` tab of the job form.
+- The points a user will earn when an applicant applies for the position.
+- The job description detailing the job position. This information is taken from the :guilabel:`Job
+  Position` tab of the job form.
+
+To see all the detailed for a job position, click the :guilabel:`More Info` button on the specific
+card. This opens the job position webpage in a new browser tab. This is what an applicant sees
+before applying for a position.
+
+.. image:: referrals/jobs.png
+   :align: center
+   :alt: The 'View Jobs' screen, displaying all current open job positions. All information is
+         displayed on the card.
+
+.. note::
+   Only published job positions will be visible in the Referrals app. To check which job positions
+   are published or not, refer to the :doc:`recruitment/new_job` documentation.
+
+Refer friends
+-------------
+
+To share a job position with someone, click the :guilabel:`Refer Friend` button on the specific job
+position card. A pre-configured :guilabel:`Send Job Offer by Mail` email pop-up window appears.
+Enter the recipient's email address in the :guilabel:`Email` field.
+
+The subject and body are populated using a default template. The subject `Job for you` appears, and
+can be modified if desired.
+
+The specific title of the job position will populate the :guilabel:`Job Position` placeholder in the
+email body. The `See Job Offer` text in the email body is an individualized tracking link link to
+the specific job position listed on the website. When the prospective employee receives the email,
+the link will send them to the job position page, where they can apply for the position, and the
+person who referred them is tracked in the Referrals application.
+
+If desired, add any text or closing salutation to the email body. When all edits have been made,
+click :guilabel:`Send Email` to send the email, or click :guilabel:`Cancel` to close the email
+pop-up window.
+
```

Comment:
Even so- these instruction work, so I am not sure what I need to add? I tried again and I can edit, add info, and send.  Since this works, I am not going to add this info, since it might confuse people if I explain hard-coded emails vs template-s since the end result is the same they can add text then send).